README.md

```diff
@@ -45,9 +45,9 @@ write, delete and modify Exif, IPTC, XMP and ICC image metadata.
 | Exiv2 Resource              | Location |
 |:------                      |:----     |
 | Project Homepage            | [https://github.com/Exiv2/exiv2](https://github.com/Exiv2/exiv2) |
-| Downloads and Documentation | [http://exiv2.dyndns.org](http://exiv2.dyndns.org:8080) |
-| BuildServer:                | [http://exiv2.dyndns.org:8080](http://exiv2.dyndns.org:8080) |
-| License (GPLv2)             | [license.txt](license.txt) |
+| Downloads and Documentation | [https://exiv2.org](https://exiv2.org) |
+| Prereleases:                | [https://prerelease.exiv2.org](https://prerelease.exiv2.org) |
```

Comment:
```suggestion
| Prereleases:                | [https://pre-release.exiv2.org](https://pre-release.exiv2.org) |
```

Reply:
Sure.  That's a typo and I'll fix it on the next "round".  There will be more typos because I'm dyslexic.